torch/_overrides.py

```diff
@@ -0,0 +1,235 @@
+"""
+Preliminary implementation of __torch_function__
+
+TODO: rewrite this in C++ for performance.
```

Comment:
This TODO is fixed now right

Reply:
Yes, this got copy/pasted over from the python implementation and I forgot to update the docstrings here. We need this for the parts of the `torch` API that are implemented in python.